### PR TITLE
add classic LB support

### PR DESCRIPTION
**Pull request by GwonsooLee**
Add classic LB support

If a user wants to deploy to classic load balancer, he/she could use `healthcheck_load_balancer` not `healthcheck_target_group`

### CODE REVIEW

File: hack/apitest.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ BUILD_DIR="build"
 TEST_DIR="test"
 URL="https://hello.devops-art-factory.com"
 
-stacks=("artd" "artd-spot" "artd-mixed" "artd-without-targetgroup")
+stacks=("artd" "artd-spot" "artd-mixed" "artd-without-targetgroup" "artd-healthcheck-lb" "artd-with-exact-values" "artd-with-exact-values" "artd-with-classic-lb")
```
Comment (asbubam) [inline hack/apitest.sh:8]: `artd-with-exact-values` has duplicated!

File: hack/apitest.sh [hunk 2]
```diff
@@ -25,6 +25,7 @@ fi
 # api test
 lastStack=""
 for stack in "${stacks[@]}"; do
+    echo $stack
```
Comment (asbubam) [inline hack/apitest.sh:28]: How about echo "stack: $stack" ?

File: hack/apitest.sh (hunk 1 above)
Comment (GwonsooLee) [reply hack/apitest.sh]: Fixed it :)

File: hack/apitest.sh (hunk 2 above)
Comment (GwonsooLee) [reply hack/apitest.sh]: I  fixed it, thanks :)

**Pull request merged**